content/file/file.go

```diff
@@ -108,6 +108,8 @@ type Store struct {
 	// value overrides the [AnnotationUnpack].
 	// Default value: false.
 	SkipUnpack bool
+	// PreserveModeBits controls whether to preserve file permissions when unpacking
+	PreserveModeBits bool
```

Comment:
nit: How about just calling it `PreservePermission` so users know that it's similar to the `tar -x --preserve-permission`?

Reply:
👍 fixed.